snowfakery/parse_recipe_yaml.py

```diff
@@ -190,12 +190,19 @@ def parse_structured_value(name: str, field: Dict, context: ParseContext) -> Def
         namespace, name = function_name.split(".")
         plugin = context.parser_macros_plugins.get(namespace)
     if plugin:
-        func = getattr(plugin, name)
-        rc = func(context, args)
-        return rc
-    else:
-        args = parse_structured_value_args(args, context)
-        return StructuredValue(function_name, args, **context.line_num(field))
+        try:
+            func = getattr(plugin, name)
+            rc = func(context, args)
```

Comment:
What's `rc` stand for? `return_context`?

Reply:
@Br4nd0R "Return code", I guess. I suppose I've been using it for 20 years automatically.